contracts/StakingPoolV2.sol

```diff
@@ -184,15 +227,14 @@ contract StakingPoolV2 is IStakingPoolV2, StakedElyfiToken {
   function initNewPool(
     uint256 rewardPerSecond,
     uint256 startTimestamp,
-    uint256 duration
+    uint32 duration
```

Comment:
Please check changing data type.

Reply:
There is no need to replace uint256 into uint32 because both use 32bytes. Storage variables are packed, but this is not true in memory and calldata.

> Among others, the ABI specification requires arguments to be padded to multiples of 32 bytes. 

Reference: https://docs.soliditylang.org/en/v0.8.4/internals/layout_in_calldata.html

Reply:
```suggestion
    uint256 duration
```